sketch_map_tool/analyses/html_gen/analyses_output_generator.py

```diff
@@ -84,12 +81,12 @@ def write_results_to_json(bbox: Bbox,
         "RESTART_LINK": f"../../analyses?bbox={bbox.get_str(mode='comma')}",
         "MAP_COORDINATES": f"{center_point[0]},{center_point[1]}",
         "PLOYGON_COORDINATES": f"[[{bbox.lat1},{bbox.lon1}],[{bbox.lat2},{bbox.lon1}],[{bbox.lat2},"
-                               f"{bbox.lon2}],[{bbox.lat1},{bbox.lon2}]]",
```

Comment:
Yes, this would happen every time `black` runs. One could disable formatting by including `# fmt: skip`
I took another approach and first assigned the string to a variable to improve readability.